package.json

```diff
@@ -39,7 +39,7 @@
     "ethereumjs-wallet": "^1.0.2",
     "execa": "^1.0.0",
     "fs-extra": "^10.1.0",
-    "ganache": "^7.5.0",
+    "ganache": "^7.7.0",
```

Comment:
Gas estimation fails even with `allowUnlimitedContractSize` when using truffle bellow `7.7.0`. It was hard to find because they did not mention it in the release page but I found it in the PR comments! ^^

https://github.com/trufflesuite/ganache/pull/3971/files#r1036263452